packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/types.ts

```diff
@@ -311,8 +311,8 @@ export type StepActionsPayload =
       step: 'CRYPTO_SELECTION'
     }
   | {
-      cryptoCurrency: CoinType
-      defaultMethod: SBFormPaymentMethod
+      cryptoCurrency?: CoinType
```

Comment:
Why is this optional?

Reply:
I think I had some issues to pass it over, will revert it not to be